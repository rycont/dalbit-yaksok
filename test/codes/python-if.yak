이름 = '홍길동'
if 이름 == '홍':
    '네 이름을 끝까지 말하렸다' 보여주기
elif 이름 == '홍길동':
    "이래도 끝까지 말하지 않느냐" 보여주기
else:
    "어서오십시오" 보여주기
