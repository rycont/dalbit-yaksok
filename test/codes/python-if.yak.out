이래도 끝까지 말하지 않느냐
